docs/coa-aks.md

```diff
@@ -0,0 +1,53 @@
+## Cromwell on AKS Instructions and Troubleshooting
+
+### Deployment Dependencies
+The CoA deployer requires the user to have Helm 3 installed locally to deploy with AKS. Use the flag "--HelmExePath HELM_PATH" to give the deployer the path to the helm exe, if no flag is passed the deployer will assume Helm is installed with Chocolatey at "C:\\ProgramData\\chocolatey\\bin\\helm.exe".
+
+### Deployment Models
+
+- ### CoA provisioned AKS account
+    Add the flag "--UseAks true" and the deployer will provision an AKS account and run its containers in AKS rather than provisioning a VM. 
+- ### Shared AKS account with CoA namespace
+    Add the flags "--UseAks true --AksClusterName {existingClusterName}", where the user has admin access to the existing AKS account, the deployer will deploy blob-csi-driver, and aad-pod-identity to the kube-system namespace, and then deploy CoA to the namespace "coa". Add the flag "--AksCoANamespace {namespace}" to override the default namespace.
+- ### Shared AKS account without developer access. 
+    If the user is required to use an AKS account, but does not have the required access, the deployer will produce a Helm chart that can then be installed by an admin or existing CI/CD pipeline. Add the flags "--UseAks true --ManualHelmDeployment". The deployer will print a postgresql command, this would typically be run on the kubernetes node to setup the cromwell user however the user will need to run this manually since the deployer won't directly access the AKS account. 
+
+    - Run the deployer with supplied flags. 
+    - Deployer will create initial resources and pause once it's time to deploy the Helm chart.
+    - Ensure the blob-csi-driver and aad-pod-identity are installed.
+    - Install the CoA Helm chart. 
+    - Run the postgresql command to create the cromwell user. 
+    - Press enter in the deployer console to finish the deployment and run a test workflow. 
+
+### Depedent Kubernetes Packages
```

Comment:
"Dependent".  Let's run a spell check on this whole file

Reply:
Done, found a couple more